CHANGELOG.md

```diff
@@ -1,5 +1,71 @@
 # Change Log
 
+## September 2023 
+
+Azure Data Studio 1.46.0 is the latest general availability (GA) release.
+
+- Release number: 1.46.0
+- Release date: September 20, 2023
+
+### What's new in 1.46.0
+
+| New Item | Details |
+| --- | --- |
+| Attach/Detach Database | Introduced support for attaching and detaching databases (Preview) |
+| Connection | Introduced support for custom cloud providers (LINK TO NEW DOC PAGE) |
```

Comment:
will this get updated with the link before merging?

Reply:
oh yeah I will fix that. I must have copied it from the Docs PR before it was finished.